draft-ietf-quic-transport.md

```diff
@@ -404,33 +419,32 @@ The fields in a Regular packet past the Common Header are the following:
   protocol.  Only present in the client-to-server direction, and if the VERSION
   flag is set.  Version Negotiation is described in {{version-negotiation}}.
 
-* DISCUSS_AND_REPLACE: Diversification Nonce: A 32-byte nonce generated by the
-  server and used only in the Server->Client direction to ensure that the server
-  is able to generate unique keys per connection.  Specifically, when using
-  QUIC's 0-RTT crypto handshake, a repeated CHLO with the exact same connection
-  ID and CHLO can lead to the same (intermediate) initial-encryption keys being
-  derived for the connection.  A server-generated nonce disallows a client from
-  causing the same keys to be derived for two distinct connections.  Once the
-  connection is forward-secure, this nonce is no longer present in packets.
-  This nonce can be removed from the packet header if a requirement can be added
-  for the crypto handshake to ensure key uniqueness.  The expectation is that
-  TLS1.3 meets this requirement.  Upon working group adoption of this document,
-  this requirement should be added to the crypto handshake requirements, and the
-  nonce should be removed from the packet format.
-
 * Packet Number: The lower 8, 16, 32, or 48 bits of the packet number, based on
   the PACKET_NUMBER_SIZE flag.  Each Regular packet is assigned a packet number
   by the sender.  The first packet sent by an endpoint MUST have a packet number
   of 1.
 
-* AEAD Data: A Regular packet's header, which includes the Common Header, and
-  the Version, Diversification Nonce, and Packet Number fields, is authenticated
+* AEAD Data: A Regular packet's Common Header and fields are authenticated,
```

Comment:
I don't think that it's accurate to say that the Regular packet's Common Header and fields are authenticated, since the "fields" contain frames and those are encrypted. Perhaps we should start calling them Regular packet header and payload, so that the Common Header and the Regular packet's Header are authenticated, and the Regular packet's payload is encrypted?